.bcr/presubmit.yml

```diff
@@ -0,0 +1,18 @@
+matrix:
+  platform:
+  - debian10
+  - ubuntu2004
+  - macos
+  - windows
```

Comment:
do we need windows here? Some stuff we are building at this point isn't necessarily Windows compatible